purchases/src/main/kotlin/com/revenuecat/purchases/interfaces/GetStorefrontCallback.kt

```diff
@@ -0,0 +1,25 @@
+//  Purchases
+//
+//  Copyright © 2025 RevenueCat, Inc. All rights reserved.
+//
+package com.revenuecat.purchases.interfaces
+
+import com.revenuecat.purchases.CustomerInfo
+import com.revenuecat.purchases.PurchasesError
+
+/**
+ * Interface to be implemented when calling [Purchases.getStorefrontCountryCode]
+ */
+interface GetStorefrontCallback {
+    /**
+     * Will be called after the call has completed.
+     * @param customerInfo [CustomerInfo] class sent back when the call has completed
```

Comment:
Wrong documentation for this method
```suggestion
     * @param storefrontCountryCode [String] containing the Store country code in ISO-3166-1 alpha2
```

Reply:
Oops, thank you!!